packages/conf-libpcre2-8/conf-libpcre2-8.2/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
```

Comment:
AFAIU, maintainer should name particular people (or organizations) who have
taken on the responsibility for maintaining a package. I don't think a link to
the opam repo issue tracker serves this end, and in any case it duplicates the
data we already have in the `bug-reports` field.

Would you be willing to maintain this package?

Reply:
Hi, for `conf-libpcre2-8`, I am just updating the dependencies and the `pkg-config` check however there is already a previous version in the repository: `conf-libpcre2-8.1`. This previous version also put the issue tracker in place of the maintainer field. I'm not sure if I have the knowledge to maintain this package for every platform.

I feel more confident putting myself as maintainer for the newly added `conf-mingw-w64-pcre2-i686` and `conf-mingw-w64-pcre2-x86_64`.

Reply:
Oh, sorry I missed that this was just update! My comment is not relevant then.